src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -131,6 +137,18 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     group5: 4,
   };
 
+  /**
+   * Should we display ads? We check:
+   * 1. The CPS `allowAdvertising` field value.
+   * 2. A value local to the STY page type.
+   * - iSite toggles are handled by the Ad container.
+   */
```

Comment:
I thought we weren't adding toggles to enable ads on individual page types, when did this change? I remember us saying it might be confusing for ops when they want to turn off ads in live and there are lots of seperate toggles for each page as well as the main ads toggle?

Reply:
This is needed to prevent Ads going out on STYs until we say so - other toggles can't be used, as Ads are already live on other page types. I imagine in future this control could be reduced to use the `allowAdvertising` field.